src/test.ts

```diff
@@ -2,57 +2,57 @@ import './index';
 import * as assert from 'assert';
 import { iter, aiter, wrap, awrap } from './ItHelper';
 
-function* numbers() {
+function* numbers(): Iterator<number> {
```

Comment:
I have added this to resolve a typescript error
```
src/test.ts:245:3 - error TS2322: Type '(this: any, ...args: any) => HIterator<number>' is not assignable to type '() => HIterator<1 | 2 | 3>'.
  Type 'HIterator<number>' is not assignable to type 'HIterator<1 | 2 | 3>'.
    Type 'number' is not assignable to type '1 | 2 | 3'.

245   n = wrap(numbers);
      ~


Found 1 error.
```

If you have a better fix, I will be happy to use it :)